### PR TITLE
Re-enable server reporting

### PR DESCRIPTION
**Pull request by otoolep**

### CODE REVIEW

**Comment by toddboom** [comment]
yeah, this looks awesome. much simpler than what i had started.

**Comment by otoolep** [comment]
Yeah, I think a `Service` is overkill for this.

File: tsdb/meta.go [hunk 1]
```diff
@@ -46,6 +46,15 @@ func (d *DatabaseIndex) Measurement(name string) *Measurement {
 	return d.measurements[name]
 }
 
+// MeasurementSeriesCounts returns the number of measurments and series currently indexed by the database.
+// Useful for reporting and monitoring.
+func (d *DatabaseIndex) MeasurementSeriesCounts() (nMeasurements int, nSeries int) {
```
Comment (otoolep) [inline tsdb/meta.go]: Named return values, for self-documentation purposes.

**Comment by toddboom** [comment]
+1

**Comment by pauldix** [comment]
LGTM :ship:

**Pull request merged**